### ISSUE TITLE
Better way to handle social (Mastodon) images

### ISSUE THREAD

**Issue by stephenmparke** [danbush/scoop#34]
They can get hilariously awkward. Can probably leave it for now, but something to consider.

![image](https://github.com/danbush/scoop/assets/1322735/42828b04-3434-4a60-ae0f-bf504b61dc86)

**Comment by stephenmparke** [comment]
@danbush feel free to move this to a different milestone (or remove from this one) as you see fit.

**Comment by danbush** [comment]
Yeah it gets funky. eventually we'll probably crop after a certain point.

**Comment by danbush** [comment]
just FYI I actually somewhat managed this by adding a max height. I think eventually we'll wanna do some cropping instead, but at least its better than it was
